site/docs/v1/tech/themes/kickstart-custom-theme.adoc

```diff
@@ -0,0 +1,400 @@
+
+```json
+{
+  "method" : "PATCH",
+  "url" : "api/theme/#{themeID}",
+  "body" : {
+    "theme" : {
+      "stylesheet" : "@{darkTheme.css}"
+    }
+  }
+}
+```
+
+This request applies our `darkTheme.css` stylesheet to the theme we created.
+
+Create a file called `createApplication.json` and copy the following into it:
+
+```json
+{
+  "method" : "POST",
+  "url" : "/api/application/#{applicationID}",
+  "body" : {
+    "application":{
+      "name" : "Kickstart App",
+      "oauthConfiguration" : {
+        "authorizedRedirectURLs" : [
+          "http://localhost:3000/oauth-redirect"
+        ],
+        "clientId" : "#{applicationID}",
+        "clientSecret" : "#{clientSecret}",
+        "logoutURL": "http://localhost:3000/logout",
+        "enabledGrants": [
+          "authorization_code",
+          "refresh_token"
+        ],
+        "requireRegistration" : "true"
+      }
+    }
+  }
+}
+```
+
+This request creates the application and configures its OAuth settings as they appear in the 5-minute guide.
+
+Create a file called `createUser.json` containing the following:
+
+```json
+{
+  "method": "POST",
+  "url": "/api/user/registration/#{userID}",
+  "body": {
+    "user": {
+      "email": "test_user@email.com",
+      "password": "1234567890"
+    },
+    "registration": {
+      "applicationId": "#{FUSIONAUTH_APPLICATION_ID}",
+      "roles": [
+        "admin"
+      ]
+    }
+  }
+}
+```
+
+This request creates a user and registers the user to the default FusionAuth application. This is necessary to login to the admin panel.
+
+Finally, create a file called `registerUser.json` containing the following:
+
+```json
+{
+  "method": "POST",
+  "url": "/api/user/registration/#{userID}",
+  "body": {
+    "registration": {
+      "applicationId": "#{applicationID}"
+    }
+  }
+}
+```
+
+This request adds a registration for the user that we just created to our custom application. This requires a separate request because our initial request used its `"registration"` field for the default application.
+
+With these files, our `kickstart` folder is complete and ready to use. The entire folder can be downloaded https://github.com/ritza-co/fusion-kickstart-custom-theme/tree/main/kickstart[here].
+
+== Modifying the Files from the 5-Minute Guide
+
+Next we'll import and modify the files from the 5-minute guide that let us launch and run our FusionAuth instance.
+
+First, download the Docker files.
+
+```bash
+curl -o docker-compose.yml https://raw.githubusercontent.com/FusionAuth/fusionauth-containers/master/docker/fusionauth/docker-compose.yml
+curl -o .env https://raw.githubusercontent.com/FusionAuth/fusionauth-containers/master/docker/fusionauth/.env
+```
+
+To enable Kickstart to run from this `docker-compose.yml` file, we must make some modifications. They are described in detail at https://fusionauth.io/docs/v1/tech/installation-guide/docker#kickstart[this link] and copied here for your convenience:
```

Comment:
Can you turn this into an include file so it can be more easily kept up to date?